### PR TITLE
dts: core-se: Disable spi0

### PR DESCRIPTION
**Pull request by nbuchwitz**
The RevPi Core SE has no ethernet phys on the piBridge. As no other devices are connected to the spi0 bus, disable it.

This was brought to attention by @l1k during the Connect 4 piControl review: https://github.com/RevolutionPi/piControl/pull/94/commits/8780dc863822661b09b83431cb3c7813e6141471

### CODE REVIEW

File: arch/arm/boot/dts/overlays/revpi-core-se-2022-overlay.dts
Comment (iluminat23) [inline arch/arm/boot/dts/overlays/revpi-core-se-2022-overlay.dts:28]: I don't remember why, but I think we left the spi0 intentionally enabled. Give me until next week, I will try to remember why.

**Comment by nbuchwitz** [comment]
Hint for myself: Merge this only after the Connect 4 PR is merged. Otherwise piControl will break on Core SE

**Review by l1k — APPROVED** [review]

File: arch/arm/boot/dts/overlays/revpi-core-se-2022-overlay.dts
Comment (nbuchwitz) [reply arch/arm/boot/dts/overlays/revpi-core-se-2022-overlay.dts:28]: Did you already had time to think about this?

Comment (iluminat23) [reply arch/arm/boot/dts/overlays/revpi-core-se-2022-overlay.dts:28]: Yeah, I can think of anything else then that picontrol would have failed otherwise. So this should be ok

Comment (nbuchwitz) [reply arch/arm/boot/dts/overlays/revpi-core-se-2022-overlay.dts:28]: So, this is good to merge then?

**Review by iluminat23 — APPROVED** [review]

**Comment by nbuchwitz** [comment]
@l1k, we should pick this for the 6.1 branch as well

**Comment by l1k** [comment]
@nbuchwitz Done. I've picked up the regdb fix as well.

**Comment by nbuchwitz** [comment]
> @nbuchwitz Done. I've picked up the regdb fix as well.

Thanks. For 5.10 see https://github.com/RevolutionPi/linux/pull/161

**Pull request merged**
